__tests__/app.test.js

```diff
@@ -101,3 +101,33 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("returns 200 status code and an array of articles sorted by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .then((response) => {
+        const articles = response.body.articles;
+        expect(response.status).toBe(200);
+        expect(Array.isArray(articles)).toBe(true);
```

Comment:
You don't need to check that `Array.isArray` here because the forEach loop you are doing will only work if it was an array